test/signatures.test.ts

```diff
@@ -31,4 +42,27 @@ describe('getSignature', () => {
     const actual = await getSignature(filePath);
     expect(actual).toStrictEqual(expected);
   });
+
+  it('binaryfile - uhash', async () => {
+    const fixturePath = path.join(__dirname, 'fixtures');
+    const filePath = path.join(fixturePath, 'dog.jpg');
+    const expected = {
+      path: filePath,
+      hashes_ffm: [{ format: 3, data: '95f0ec135ee38f07e07857d6' }],
+    };
+    const actual = await getSignature(filePath, 'uhash');
+    expect(actual).toStrictEqual(expected);
+  });
+
+  it('should throw exception if unsupported hashType is specified', async () => {
+    expect.assertions(1);
+    const fixturePath = path.join(__dirname, 'fixtures');
+    const filePath = path.join(fixturePath, 'dog.jpg');
+    const expected = new Error('Unsupported hashType non-existing-hash-type');
+    try {
+      await getSignature(filePath, 'non-existing-hash-type');
+    } catch (err) {
+      expect(err).toEqual(expected);
```

Comment:
Prefered way to deal with exception in jests would be

```using expect(() => ...).rejects.toThrow(expected)...```

e.g.
```
    const invalidHashAlgorithmType = 'non-existing-hash-type' as SignatureHashAlgorithm;
    expect(
      async () =>
        await getSignaturesByAlgorithm([filePath], invalidHashAlgorithmType),
    ).rejects.toThrow(expected);
 ```
 
cast required as the type system will complain TS wise, even so we have the test passing because it will throw in runtime as desired.

would be nice to have

```
export type SignatureHashAlgorithm = 'dubhash' | 'uhash';
```

added into `lib/types.ts`